classes/mail/variables/IssueEmailVariable.php

```diff
@@ -67,4 +71,33 @@ protected function getIssueUrl(): string
             $this->issue->getBestIssueId()
         );
     }
+
+    protected function getIssueToc(): string
+    {
+        $request = Application::get()->getRequest();
+
+        $submissions = Repo::submission()->getCollector()
+            ->filterByContextIds([$this->getContext()->getId()])
+            ->filterByIssueIds([$this->issue->getId()])
```

Comment:
I think you'll also need a filter on submission status:
```php
->filterByStatus([PKPSubmission::STATUS_PUBLISHED])
```
...and the right ordering:
```php
->orderBy(\APP\submission\Collector::ORDERBY_SEQUENCE, \APP\submission\Collector::ORDER_DIR_ASC)
```

Reply:
Thanks, updated .